cmake/GzCodeCoverage.cmake

```diff
@@ -141,10 +141,14 @@ FUNCTION(gz_setup_target_for_coverage)
     # Capturing lcov counters and generating report
     COMMAND ${LCOV_PATH} ${_branch_flags} -q --no-checksum
       --directory ${PROJECT_BINARY_DIR} --capture
-      --output-file ${_outputname}.info 2>/dev/null
+      --ignore-errors gcov,gcov
```

Comment:
Does it make sense instead of ignoring categories doing it per files / per directories, something like:
 ```
  --exclude '/usr/include/*' \
  --exclude '*_TEST.cc' \
```

Reply:
we use `--remove` with lots of path wildcards on line 152. I'm not sure what the best approach is here. I think we would need to test with more packages as well so see if more `--ignore-errors` categories are needed